contents/overlay/dynamic-islands/simple-dynamic-island/index.tsx

```diff
@@ -4,92 +4,118 @@ import {
   Link,
   Motion,
   useDisclosure,
+  useBreakpointValue,
   useOutsideClick,
 } from "@yamada-ui/react"
 import type { FC } from "react"
 import { useRef } from "react"
 
-const SimpleDynamicIsland: FC = () => {
-  const { isOpen, onOpen, onClose } = useDisclosure()
+const DynamicIsland: FC = () => {
   const ref = useRef<HTMLDivElement>(null)
-
-  const handleExpand = () => {
-    if (isOpen) {
-      onClose()
-    } else {
-      onOpen()
-    }
-  }
+  const { isOpen, onClose, onOpen, onToggle } = useDisclosure()
+  const width = useBreakpointValue({
+    base: 380,
+    md: 320,
+  })
 
   useOutsideClick({
     ref,
     handler: () => {
-      if (isOpen) {
-        onClose()
-      }
+      if (isOpen) onClose()
     },
   })
 
   return (
-    <Center
-      as={Motion}
-      initial={{ width: 200, height: 60 }}
+    <Motion
+      initial={{
+        width: 250,
+        height: 40,
+        paddingInline: "0.5rem",
+        paddingBlock: "0.5rem",
+      }}
       animate={{
-        width: isOpen ? 300 : 200,
-        height: 60,
-        transition: { duration: 0.5 },
+        width: isOpen ? width : 250,
+        height: isOpen ? 60 : 40,
+        paddingInline: isOpen ? "1rem" : "0.5rem",
+        paddingBlock: isOpen ? "1rem" : "0.5rem",
       }}
-      onClick={handleExpand}
+      onHoverStart={onOpen}
+      onHoverEnd={onClose}
+      onClick={onToggle}
       bg={["black", "blackAlpha.600"]}
-      shadow="xl"
+      shadow="2xl"
+      display="flex"
       justifyContent="space-between"
+      alignItems="center"
       cursor="pointer"
       position="fixed"
-      top="md"
+      bottom="lg"
       left="50%"
       transform="translateX(-50%)"
-      rounded="lg"
-      px="md"
+      rounded="full"
       ref={ref}
+      transition={{ type: "spring", stiffness: 300, damping: 20 }}
     >
-      <Avatar size="sm" />
+      <Avatar
+        as={Motion}
+        initial={{ height: 24, width: 24 }}
+        animate={{
+          height: isOpen ? 42 : 24,
+          width: isOpen ? 42 : 24,
+        }}
+      />
 
-      {isOpen ? (
+      {isOpen && (
         <Motion
-          initial={{ opacity: 0 }}
-          animate={{ opacity: 1 }}
-          transition={{ duration: 0.5, delay: 0.2 }}
+          initial={{ opacity: 0, translateY: -10 }}
+          animate={{ opacity: 1, translateY: 0 }}
+          transition={{ duration: 0.4, delay: 0.1 }}
         >
           <Center gap="md">
```

Comment:
I think it would look better if you added `px="md"` to `<Center gap="md">`. What do you think?

Reply:
Fixed.